torch/linalg/__init__.py

```diff
@@ -177,6 +180,12 @@
     >>> torch.linalg.det(a)
     tensor(0.0934)
 
+    >>> out = torch.Tensor()
```

Comment:
torch.tensor(), the legacy torch.Tensor() should be considered deprecated

Reply:
fixed by dddc37b924.
`torch.tensor()` alone doesn't work, it requires a shape. I ended up using `torch.empty(0)` which looks clearer than `torch.tensor([])` to me.